api/src/services/member/member.service.ts

```diff
@@ -189,6 +189,7 @@ export class MemberService {
       lastName: member.lastname,
       chatdefault: member.chatdefault,
       primary_role_id: member.primary_role_id,
+      lastAccess: member.last_activity,
```

Comment:
Normal users will not need this informations, just anyone with admin right for that house. Can we remove this line?